lib/glimesh_web/templates/layout/app.html.eex

```diff
@@ -1,8 +1,9 @@
+<!-- app.html.eex -->
```

Comment:
I've found it saves debugging effort later on to wrap shared html code snippets in a comment that describes where it comes from.